doc/modules/cross_validation.rst

```diff
@@ -467,44 +367,180 @@ Here is a usage example::
 validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
+Cross-validation iterators with stratification based on class labels.
+=====================================================================
+
+In some datasets the number of samples per class vary based on the class label.
+They are referred to commonly as datasets with imbalance in classes. In such
+cases, stratified sampling must be done to pick the samples for
+cross-validation. This avoids the class imbalance from affecting the
+cross-validation results.
+
+The :class:`StratifiedKFold` and :class:`StratifiedShuffleSplit` do
+stratification to ensure that the proportion of samples from each class is
+same before cross-validating the stratified samples by k-fold cross-validation
```

Comment:
I don't think "before", rather they ensure proportions are similar in each train and test portion.